protocol/src/main/proto/LiiklusService.proto

```diff
@@ -57,9 +80,9 @@ message SubscribeRequest {
 
     string group = 2;
 
-    uint32 groupVersion = 4;
+    uint32 group_version = 4;
 
-    AutoOffsetReset autoOffsetReset = 3;
+    AutoOffsetReset auto_offset_reset = 3;
```

Comment:
is that a good practice to go snake_case?

Reply:
Yes : https://developers.google.com/protocol-buffers/docs/style#message-and-field-names

This has the advantage of generating python-friendly names while preserving what was there for Java and other languages.